assets/grafana/deployment.yaml

```diff
@@ -95,6 +102,9 @@ spec:
         - mountPath: /etc/grafana
           name: grafana-config
           readOnly: false
+        - mountPath: /tmp
+          name: tmp-plugins
+          readOnly: false
```

Comment:
This [issue](https://github.com/openshift/cluster-monitoring-operator/pull/1556/files#r802548538) fixed here